search_indices/backends/faceted_attribute_backend.py

```diff
@@ -28,13 +26,23 @@ class FacetedAttributeBackend(BaseFilterBackend, FilterBackendMixin):
     faceted_search_param = "facet_attribute"
 
     @staticmethod
-    def get_attributes(documents: List[Type[Document]] = None) -> list:
+    def get_attributes(
+        documents: list[type[Document]] = None, authenticated=True
```

Comment:
`authenticated=True`

* defaulting optional flag related to authentication to true is  a bit meh
* "authenticated" does not tell the caller what it does, compared for example to something like "include_information_system"

Reply:
While you're at it, this is a boolean trap. Use `def get_attributes(documents: ..., *, authenticated=True)` instead (or whatever name/value you settle on)

Reply:
Valid argument. I'd go with `include_information_system`